test/ingest_search/ingest_search_test.go

```diff
@@ -0,0 +1,235 @@
+				} else {
+					testenvInstance.Log.Info("Sync Search results:", "prettyResults", string(prettyResults))
+				}
+
+				return standalone.Status.Phase
+			}, deployment.GetTimeout(), PollInterval).Should(Equal(splcommon.PhaseReady))
+
+			Eventually(func() splcommon.Phase {
+				podName := fmt.Sprintf("splunk-%s-standalone-0", deployment.GetName())
+				searchString := "index=_internal GUID component=ServerConfig"
+
+				// Perform a simple search
+				sid, reqErr := testenv.PerformSearchReq(podName, searchString, deployment)
+				if reqErr != nil {
+					testenvInstance.Log.Error(reqErr, "Failed to execute search on pod", "pod", podName, "searchString", searchString)
+					return splcommon.PhaseError
+				}
+				testenvInstance.Log.Info("Got a search with sid", "sid", sid)
+
+				// Check SID status until done
+				searchStatusResult, statusErr := testenv.GetSearchStatus(podName, sid, deployment)
+				if statusErr != nil {
+					testenvInstance.Log.Error(statusErr, "Failed to get search status on pod", "pod", podName, "sid", sid)
+					return splcommon.PhaseError
+				}
+				testenvInstance.Log.Info("Search status:", "searchStatusResult", searchStatusResult)
+
+				// Get SID results
+				searchResultsResp, resErr := testenv.GetSearchResults(podName, sid, deployment)
+				if resErr != nil {
+					testenvInstance.Log.Error(resErr, "Failed to get search results on pod", "pod", podName, "sid", sid)
+					return splcommon.PhaseError
+				}
+
+				// Display results for debug purposes
+				prettyResults, jsonErr := json.MarshalIndent(searchResultsResp, "", "    ")
+				if jsonErr != nil {
+					testenvInstance.Log.Info("Failed to generate pretty json", jsonErr)
+				} else {
+					testenvInstance.Log.Info("Search results:", "prettyResults", string(prettyResults))
+				}
+
+				return standalone.Status.Phase
+			}, deployment.GetTimeout(), PollInterval).Should(Equal(splcommon.PhaseReady))
+		})
+	})
+
+	Context("Standalone deployment (S1)", func() {
+		It("can ingest custom data to new index and search", func() {
+
+			standalone, err := deployment.DeployStandalone(deployment.GetName())
+			Expect(err).To(Succeed(), "Unable to deploy standalone instance ")
+
+			// Wait for standalone to be in READY Status
+			testenv.StandaloneReady(deployment, deployment.GetName(), standalone, testenvInstance)
+
+			// Verify splunk status is up
+			Eventually(func() splcommon.Phase {
+				podName := fmt.Sprintf("splunk-%s-standalone-0", deployment.GetName())
+
+				var statusCmd strings.Builder
+				splunkBin := "/opt/splunk/bin/splunk"
+				username := "admin"
+				password := "$(cat /mnt/splunk-secrets/password)"
+				splunkCmd := "status"
+
+				fmt.Fprintf(&statusCmd, "%s %s -auth %s:%s", splunkBin, splunkCmd, username, password)
+				command := []string{"/bin/bash"}
+				stdin := statusCmd.String()
+				statusCmdResp, stderr, err := deployment.PodExecCommand(podName, command, stdin, false)
```

Comment:
Is there a http client we can use in the test repo?

Reply:
No not yet.  This one is a CLI command too, so http client wouldn't help here. :)

Reply:
Oh I meant extracting fields and storing might be easier with a REST API call vs CLI + string manipulation. But it is something which we can look at in the future.